db/dao/sector_dao.py

```diff
@@ -16,6 +17,50 @@ def create_sector(sector, commit=True):
 
         return sector_db
 
+    @staticmethod
+    def get_total_sectors_count():
+        """ Returns the total sectors created in the
+        daatabase at the moment.
+        Used to increment order.
+        """
+        return DB().session.query(Sector).count()
+
+    @staticmethod
+    def create_sectors_division(sector_size, total_size):
+        """Creates all the possible sector records in the
+        database
+        :param sector_size: Specifies the Sector size
+        :param total_size: Specifies the total disk size
+        """
+        total_sectors = sector_size // total_size
+        for sector in range(total_sectors):
```

Comment:
So, the order field is used by a file to know which of its data belongs where. It is not used as a global order field.

Let's say a file has "Hello World" with a sector size of 5.
It will have two sectors: "Hello " sector with order 1, and "World" sector with order 2.

There might be another file which has "Bye World".
It will also have two sectors: "Bye W" sector with order 1, and "orld" sector with order 2.